app/controllers/events_controller.rb

```diff
@@ -13,7 +12,7 @@ def create
 			render :text => "null"
 			return
 		end
-		@comment = Event.new(:team_id => @team[:id], :match_id => @match[:id], :goal_id => @goal[:id], :success => params[:success], :start_time => params[:start], :end_time => params[:end])
+		@comment = Event.new(:team_id => @team[:id], :match_id => @match[:id], :goal => params[:goal], :success => params[:success], :start_time => params[:start], :end_time => params[:end], :extra => params[:extra])
```

Comment:
Should this be `@event` instead of `@comment`?

Reply:
Ok; fixed